lib/internal/repl.js

```diff
@@ -55,15 +55,26 @@ function createRepl(env, opts, cb) {
   }
 
   const repl = REPL.start(opts);
-  if (opts.terminal && env.NODE_REPL_HISTORY !== '') {
```

Comment:
That makes sense to me and seems like it would be easier to maintain. That work for you @Fishrock123?